src/go/rpk/pkg/cli/cmd/cluster/maintenance/maintenance.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package maintenance
+
+import (
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/cli/cmd/common"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewMaintenanceCommand(fs afero.Fs) *cobra.Command {
+	var (
+		adminURL       string
+		adminEnableTLS bool
+		adminCertFile  string
+		adminKeyFile   string
+		adminCAFile    string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "maintenance",
+		Short: "Interact with cluster maintenance mode.",
```

Comment:
```suggestion
		Short: "Toggle a node's maintenance mode.",
```
Since it acts on specific nodes, not the whole cluster by default, right?

Reply:
That's right. In the future it will be more explicitly cluster-oriented, but for now I think your choice of working is more precise.